charts/istio-endpoints/templates/sidecar.yaml

```diff
@@ -0,0 +1,62 @@
+{{- if .Values.sidecar.enabled }}
+apiVersion: networking.istio.io/v1beta1
+kind: Sidecar
```

Comment:
the expectation is that in staging/prod this would be disabled, because this should be injected by the Istio controller, right? This is for local dev where you don't have the operator/controller installed?

Reply:
actually that looks like it's not true? I guess I don't understand how this sidecar interacts with the sidecar that's already being created by istio